src/main/kotlin/no/nav/syfo/service/SykmeldingService.kt

```diff
@@ -176,21 +181,25 @@ class SykmeldingService constructor(
                     PAPIRSM_MAPPET.labels("feil").inc()
                     log.warn("Noe gikk galt ved mapping fra OCR til sykmeldingsformat: ${e.message}, {}", fields(loggingMeta))
 
-                    val sakId = sakClient.finnEllerOpprettSak(sykmeldingsId = sykmeldingId, aktorId = aktorId, loggingMeta = loggingMeta)
-
-                    val oppgave = oppgaveService.opprettOppgave(aktoerIdPasient = aktorId, sakId = sakId,
-                            journalpostId = journalpostId, gjelderUtland = false, trackingId = sykmeldingId, loggingMeta = loggingMeta)
-
-                    if (!oppgave.duplikat) {
-                        log.info("Opprettet oppgave med {}, {} {}",
-                                StructuredArguments.keyValue("oppgaveId", oppgave.oppgaveId),
-                                StructuredArguments.keyValue("sakid", sakId),
-                                fields(loggingMeta)
-                        )
-                        PAPIRSM_OPPGAVE.inc()
+                    val papirSmRegistering = mapOcrFilTilPapirSmRegistrering(
+                            journalpostId = journalpostId,
+                            fnr = fnr,
+                            aktorId = aktorId,
+                            dokumentInfoId = dokumentInfoId,
+                            datoOpprettet = datoOpprettet,
+                            sykmeldingId = sykmeldingId,
+                            sykmelder = sykmelder,
+                            ocrFil = ocrFil
+                    )
+
+                    val duplikatOppgave = oppgaveService.duplikatOppgave(
+                            journalpostId = journalpostId, trackingId = sykmeldingId, loggingMeta = loggingMeta)
+
+                    if (!duplikatOppgave) {
+                        kafkaproducerPapirSmRegistering.send(ProducerRecord(sm2013SmregistreringTopic, papirSmRegistering.sykmeldingId, papirSmRegistering))
```

Comment:
Her kan vi gjerne legge på det som må til for at sending ikke skal feile silently. Det gjelder forresten også de andre stedene i appen der vi skriver til kafka, men hvis du synes denne pr-en er stor nok som den er kan vi fikse det i en egen pr :)